addons/web/static/src/views/list/list_renderer.js

```diff
@@ -184,6 +187,37 @@ export class ListRenderer extends Component {
         });
     }
 
+    /**
+     * @returns {Array} of name of 'Properties' fields
+     */
+    getPropertiesFieldNames() {
+        const propertiesFieldNames = [];
+        for(const record of Object.values(this.props.list.records)) {
+            const recordFields = Object.values(record.fields);
+            for(const field of recordFields) {
+                if(field.type === 'properties' && !propertiesFieldNames.includes(field.name)) {
```

Comment:
what if you have 2 properties field and the name of the first include the name of the other ? (e.g. `properties` and `company_properties`)

Reply:
Mm, I don't think this would be an issue here? This includes was meant to check over an array of String, not to check 2 Strings, I'll rename propertiesFieldName and propertiesFieldNames to be less similar.

Reply:
ah ok sorry :)